.github/wiki/Contribution.md

```diff
@@ -0,0 +1,242 @@
+
+Example
+```elixir
+Generator.replace_content(
+  "test/test_helper.exs",
+  """
+  ExUnit.start()
+  """,
+  """
+  ExUnit.start(capture_log: true)
+  """
+)
+```
+
+`delete_content(file_path, anchor)`
+
+Find and remove specified content of an existing file.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| file_path | String | File path |
+| anchor | String | Content to delete |
+
+Example
+```elixir
+Generator.delete_content(
+  "assets/js/app.js",
+  """
+  // We import the CSS which is extracted to its own file by esbuild.
+  // Remove this line if you add a your own CSS build pipeline (e.g postcss).
+  import "../css/app.css"
+
+  """
+)
+```
+
+`inject_content(file_path, anchor, content)`
+
+Inject a specified content below a specified content of an existing file.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| file_path | String | File path |
+| anchor | String | Content to find |
+| content | String | Content to inject |
+
+Example
+```elixir
+Generator.inject_content(
+  support_case_path,
+  """
+    use ExUnit.CaseTemplate
+  """,
+  """
+
+    alias Ecto.Adapters.SQL.Sandbox
+  """
+)
+```
+
+`append_content(file_path, content)`
+
+Append a specified content to the end of an existing file.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| file_path | String | File path |
+| content | String | Content to append |
+
+Example
+```elixir
+Generator.append_content(
+  "assets/css/_variables.scss",
+  """
+  Content
+  """
+)
+```
+
+`inject_mix_dependency(dependency)`
+
+Inject a new mix dependency into `mix.exs` file.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| dependency | Tuple | Item of dependency |
+
+`dependency` can also be a list of tuple in case of multiple items.
+
+```elixir
+Generator.inject_mix_dependency(
+  {:credo, latest_package_version(:credo), only: [:dev, :test], runtime: false}
+)
+```
+
+`make_directory(path, touch_directory \\ true)`
+
+Create a new directory.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| path | String | Path to create the new directory |
+| touch_directory | Boolean | Create `.keep` file inside the directory |
+
+Example
+```elixir
+Generator.make_directory("assets/css/vendor/", false)
+```
+
+`create_file(path, content)`
+
+Create a new file with specified content.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| path | String | Path to create the new file |
+| content | String | Content to put into |
+
+Example
+```elixir
+Generator.create_file(
+  "assets/css/_variables.scss",
+  """
+  Content
+  """
+)
+```
+
+`print_log(prefix, content \\ "")`
+
+Print log to console.
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| prefix | String | Prefix of message to display |
+| content | String | Content to display |
+
+Example
+```elixir
+Generator.print_log("* applying ", inspect(__MODULE__))
+```
+
+# Run unit test on local
+
+`mix test`
+
+# Apply the whole template on local
+
+See [[Installation]]
```

Comment:
Note: We will change this link `[[Installation]]` once we merged the other PR #171

Reply:
Fixed in c50e06b